frontend/src/App.tsx

```diff
@@ -73,26 +69,6 @@ const App = (): React.ReactElement => {
                   path={Routes.GLOBAL_FORMS_PAGE}
                   component={GlobalFormsPage}
                 />
-                <PrivateRoute
-                  exact
-                  path={Routes.CREATE_ENTITY_PAGE}
-                  component={CreatePage}
-                />
-                <PrivateRoute
-                  exact
-                  path={Routes.UPDATE_ENTITY_PAGE}
-                  component={UpdatePage}
-                />
-                <PrivateRoute
-                  exact
-                  path={Routes.DISPLAY_ENTITY_PAGE}
-                  component={DisplayPage}
-                />
-                <PrivateRoute
-                  exact
-                  path={Routes.EDIT_TEAM_PAGE}
-                  component={EditTeamInfoPage}
-                />
                 <PrivateRoute
```

Comment:
Do we need this Hooks page either?

Reply:
Wasn't sure if it's worth keeping for demo? Don't think we use them yet; admittedly not sure we will